docs/ReleaseInstructions.md

```diff
@@ -0,0 +1,84 @@
+# Release process
+
+## Prior to release: 1 week out
+
+* Check all dependency version ranges.
+  * Ensure all dependencies are not capped by major version, unless the reason is documented inline.
+    * Example of major version cap: `scikit-learn<2`
```

Comment:
In quick reading this looks like an example how to do it. Maybe emphasize avoiding this? Examples usually read as instructions. (also see other places where examples are non-prescriptive).
```markdown
* Ensure all dependencies are not capped by major version, unless the reason is documented inline.
  * ❎ Avoid - major version cap: `scikit-learn<2`
```

Reply:
Good suggestion! Though we might want to do the same to our release instruction in our main repo, as this is just copied from there :)